fix-txfonts-precapprox
The txfonts exchange precapprox and succapprox in comparison to the AMS fonts
and to their own negated variants. This patch fixes that by exchanging the
definitions for those two.
Norbert Preining <preining@logic.at>
---
 texmf-dist/tex/latex/txfonts/txfonts.sty |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: texlive-base-2007/texmf-dist/tex/latex/txfonts/txfonts.sty
===================================================================
--- texlive-base-2007.orig/texmf-dist/tex/latex/txfonts/txfonts.sty	2008-01-22 07:47:20.000000000 +0100
+++ texlive-base-2007/texmf-dist/tex/latex/txfonts/txfonts.sty	2008-01-22 07:47:34.000000000 +0100
@@ -579,8 +579,8 @@
 \re@DeclareMathSymbol{\thicksim}{\mathrel}{AMSb}{115}
 \re@DeclareMathSymbol{\thickapprox}{\mathrel}{AMSb}{116}
 \re@DeclareMathSymbol{\approxeq}{\mathrel}{AMSb}{117}
-\re@DeclareMathSymbol{\succapprox}{\mathrel}{AMSb}{118}
-\re@DeclareMathSymbol{\precapprox}{\mathrel}{AMSb}{119}
+\re@DeclareMathSymbol{\succapprox}{\mathrel}{AMSb}{119}
+\re@DeclareMathSymbol{\precapprox}{\mathrel}{AMSb}{118}
 \re@DeclareMathSymbol{\curvearrowleft}{\mathrel}{AMSb}{120}
 \re@DeclareMathSymbol{\curvearrowright}{\mathrel}{AMSb}{121}
 \re@DeclareMathSymbol{\digamma}{\mathord}{AMSb}{122}
